pipeline/scripts/4/3/tier-2/test-deploy_okr_bz_1994827.sh

```diff
@@ -0,0 +1,49 @@
+#! /bin/sh
+echo "Beginning Red Hat Ceph RPM based upgrade testing."
```

Comment:
Kindly fix the doc string... I believe it is `Deploy OKR testing`

Reply:
file name should include "upgrade"
it is ***upgrade_okr_bz_1994827.sh

and the doc string could be modified accordingly


and hence modify the doc string accordingly